src/index.ts

```diff
@@ -61,8 +61,13 @@ export class GistBox {
   public async update (updates: Updates) {
     const gist = await this.get()
     const filename = Object.keys(gist.data.files)[0]
+    const defaultContent = gist.data.files[filename].content || ''
     return request('PATCH /gists/:gist_id', {
-      files: { [filename]: updates },
+      files: { [filename]: {
+        content: updates.content || defaultContent,
+        filename: updates.filename || filename
```

Comment:
The interface from `@octokit/request` now constrains that there values cannot be `undefined`, so I added default values for them.
No behavior changes, just for the correct type of interface.